### PR TITLE
Virt guide proofdrop checks

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/libvirt_configuration_virsh.xml [hunk 1]
```diff
@@ -1958,7 +1958,7 @@ domid=3: nr_frames=3, max_nr_frames=40
   <sect2 xml:id="virsh-kvm-zseries-crypto-cover">
    <title>What is covered</title>
    <para>
-    You will learn how to dedicate a crypto adapter and domains on a &zseries;
+    You will learn how to dedicate a crypto adapter and domains on an &zseries;
```
Comment (dariavladykina) [inline xml/libvirt_configuration_virsh.xml:1961]:
```suggestion
    You will learn how to dedicate a crypto adapter and domains on a &zseries;
```
```suggestion
    You will learn how to dedicate a crypto adapter and domains on an &zseries;
```

Comment (dariavladykina) [reply xml/libvirt_configuration_virsh.xml:1961]: Is it pronounced as [zet-systems}?

Comment (dariavladykina) [reply xml/libvirt_configuration_virsh.xml:1961]: Is it pronounced as [zet-systems}?

File: xml/libvirt_host.xml [hunk 2]
```diff
@@ -411,7 +411,7 @@ Connection successfully activated (master waiting for slaves) \
       <para>
        Select the interface that the VLAN device should connect to below
        <guimenu>Real Interface for VLAN</guimenu>. If the desired interface
-       does not appear in the list, first set up this interface without an IP
+       does not appear in the list, first setup this interface without an IP
```
Comment (dariavladykina) [inline xml/libvirt_host.xml:414]:
```suggestion
       does not appear in the list, first set up this interface without an IP
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some corrections. Thanks!

File: xml/libvirt_configuration_virsh.xml (hunk 1 above)
Comment (tbazant) [reply xml/libvirt_configuration_virsh.xml:1961]: the entity is "IBM z-series"

**Review by tbazant — COMMENTED** [review]
Suggestions resolved

**Pull request merged**